docs/package-inclusion-guide.md

```diff
@@ -0,0 +1,55 @@
+# Azure Cloud Shell Package Inclusion Guide
+
+## Background
+
+There are millions (or potentially billions) of Linux packages. This guide goes through the principles of what the Azure Cloud Shell team uses to consider what gets included in the service.
+
+
+
+## General Principles
+
+1. Useful for Azure users and administrators 
+2. Available on [Azure Linux Operating System](https://github.com/microsoft/azurelinux) 
+3. Vulnerability free 
+    - No unfixed CVEs for the package on Azure Linux OS past 30 days 
+4. Appropriate Licensing 
+    - Azure Cloud Shell should be allowed to both consume and distribute the package 
+5. Well-maintained and supported 
+    - Package should NOT be deprecated or archived 
+    - Package repo should have a contribution/commit within the last 120 days 
+    - The package should be of a stable version. The package should not be in alpha or beta version. 
+
+6. Reasonable package size 
+
+> [!NOTE]
+> After these principles have been satisfied, the Azure Cloud Shell team reserves the right to make final decisions on package inclusion at their discretion.
+
+## Discussion of Principles
+
+1. Useful for Azure users and administrators 
+
+All packages shipped by Azure Cloud Shell should be useful to Azure users. This principle is fundamental to the core of what we ship. If the package is not believed to be used widely, it will not be shipped in Cloud Shell. 
+
+2. Available on Azure Linux OS 
+
+Due to secure supply chain compliance that Cloud Shell abides to, we are required to consume packages from secure sources. The guidance from the supply chain team is to ONLY consume from Azure Linux OS. When it’s available on Azure Linux, we have the confidence that the binary has been built by a trusted Microsoft source. To find if the package is available in Azure Linux OS, please follow the documentation provided here: [Add a package into Cloud Shell](./add-package-into-cloudshell.md#requesting-the-package)
```

Comment:
Should we add that it must be in Azure Linux 2.0 right now and that we will eventually switch to Azure Linux 3.0?